test/baseai.test.ts

```diff
@@ -1,98 +1,5 @@
-import AinftJs from "../src/ainft";
```

Comment:
why were these test cases deleted?

Reply:
Sorry, In [another PR](https://github.com/ainft-team/ainft-js/pull/234), that test file was deleted(renamed).
Now, I reverted these test cases. thanks